string table
string data
---
int64  id